appending to LD_LIBRARY_PATH, rather than rewriting it
updated: Denis Kristak (INUITS)
diff -ruN sra-tools-3.0.0.orig/build/ld.linux.dlib.sh sra-tools-3.0.0/build/ld.linux.dlib.sh
--- sra-tools-3.0.0.orig/build/ld.linux.dlib.sh	2022-02-10 14:00:25.000000000 -0800
+++ sra-tools-3.0.0/build/ld.linux.dlib.sh	2022-08-18 12:09:51.955470000 -0700
@@ -83,8 +83,7 @@
     done
 
     # update LD_LIBRARY_PATH
-    unset LD_LIBRARY_PATH
-    export LD_LIBRARY_PATH="$LDIRS:$XDIRS"
+    export LD_LIBRARY_PATH="$LDIRS:$XDIRS:$LD_LIBRARY_PATH"
 
     # tack on libraries, finding as we go
     for LIB in $LIBS
@@ -308,4 +307,4 @@
     echo "$MSG" >> $LOGFILE
 
     "$BUILD_DIR/scm-collect.sh" "$OBJS" "$SLIBS" | sort -u > "$TARG.md5"
-fi
\ No newline at end of file
+fi
diff -ruN sra-tools-3.0.0.orig/build/ld.linux.exe.sh sra-tools-3.0.0/build/ld.linux.exe.sh
--- sra-tools-3.0.0.orig/build/ld.linux.exe.sh	2022-02-10 14:00:25.000000000 -0800
+++ sra-tools-3.0.0/build/ld.linux.exe.sh	2022-08-18 12:10:58.180786000 -0700
@@ -96,8 +96,7 @@
     done
 
     # update LD_LIBRARY_PATH
-    unset LD_LIBRARY_PATH
-    export LD_LIBRARY_PATH="$LDIRS:$XDIRS"
+    export LD_LIBRARY_PATH="$LDIRS:$XDIRS:$LD_LIBRARY_PATH"
 
     # tack on libraries, finding as we go
     for LIB in $LIBS